pwa/cypress/plugins/cy-ts-preprocessor.js

```diff
@@ -0,0 +1,26 @@
+const wp = require('@cypress/webpack-preprocessor')
```

Comment:
What is this file doing? adding webpack to cypress?

Reply:
I ran the Cypress tests and this file was autogenerated. It adds webpack and other features for TS support, such as transpiling TS and using JS/TS for Cypress testing. Should I remove it?

Reply:
I was just curious as to what it was. If it was auto-generated I'm inclined to leave it unless @bmitchinson or @meghanmc have more insight.

Reply:
I'm not sure what this is. If I run the cypress tests (via `npm run cy:run`), the file isn't generated for me. I think it should be removed but @bmitchinson should confirm.

Reply:
It's required for some coverage stuff ~ autogen'd